simple-todo-tdd/client/__tests__/TodoUIActions.test.js

```diff
@@ -0,0 +1,12 @@
+import { shallow } from 'enzyme';
+
+describe('<App /> shallow rendering', () => {
+  it('adds one item in todo list on clicking add', () => {
```

Comment:
The description should start with **should** as it expects to do what has been said, and not start with **does** which implies it will always do what it says.